src/predict.Rmd

```diff
@@ -133,6 +133,32 @@ As mentioned above, the predicted posterior probabilities can be accessed via th
 head(getPredictionProbabilities(pred))
 ```
 
+### Extract Standard Errors
+
+Some Learners provide standard errors for predictions, which can be accessed in mlr. An overview is given by calling the function[&listLearners]. In order to include learners from packages which are not installed in the overview, one has to set ``check.packages``  as ``FALSE``.
+
+```{r, results='hide', warning=FALSE}
+listLearners("regr", check.packages = FALSE)
+```
```

Comment:
Hi,
Thank you very much for all the notes! We add the chunk options because we get a lot of warnings as we didn't install all packages. Now I removed it.